### PR TITLE
feat: LH - removed property-manager from microservice

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes/Fixes/Resolves

# Explain the feature/fix

1. Removed property-manager from lighthouse micro service
2. Fixed all eslint errors
    - Disabled `default export` rule because its completely havoc if we use default everywhere
    - Disabled `class methods should be static if not using this.` because we are using singleton pattern

## Does this PR introduce a breaking change

Yes

Property manager Graphql queries wont be available anymore

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by ghanlohar — APPROVED** [review]
Looks good to me.

File: packages/lighthouse-service/.eslintrc.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = {
   },
   plugins: ["@typescript-eslint"],
   rules: {
-    "max-len": ["error", { code: 130 }],
+    "max-len": [ "error", { code: 130 } ],
+    "import/prefer-default-export": "off"
```
Comment (riginoommen) [inline packages/lighthouse-service/.eslintrc.js:18]:
```suggestion
```
https://airbnb.io/javascript/#:~:text=back%20to%20top-,Modules,-10.1%20Always%20use

Comment (akhilmhdh) [reply packages/lighthouse-service/.eslintrc.js:18]: https://blog.piotrnalepa.pl/2020/06/26/default-exports-vs-named-exports/

A lot more articles 😅 . It's a nightmare

**Review by riginoommen — APPROVED** [review]

File: packages/lighthouse-service/src/helpers.ts [hunk 2]
```diff
@@ -62,47 +70,70 @@ class LHCIHelper {
...
-        });
-        return scores;
-      }, (error: any) => {
-        throw new Error(error);
-      });
+          return scores;
+        },
+        (error: any) => {
+          throw new Error(error);
+        },
```
Comment (deshmukhmayur) [inline packages/lighthouse-service/src/helpers.ts:135]: You can just put this inside a `.catch()` instead.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/lighthouse-service/src/helpers.ts (hunk 2 above)
Comment (akhilmhdh) [reply packages/lighthouse-service/src/helpers.ts]: Done. Also added fetch res.ok check as catch wont work like that in fetch

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**